queries.sql

```diff
@@ -1,17 +1,52 @@
 /*Queries that provide answers to the questions from all projects.*/
 
 SELECT name FROM animals WHERE name LIKE '%mon';
-
 SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' and '2019-12-31';
-
 SELECT name FROM animals WHERE neutered IS TRUE AND escape_attempts < 3;
-
 SELECT date_of_birth FROM animals WHERE name IN ('Agumon','Pikachu');
-
 SELECT name, escape_attempts FROM animals WHERE weigth_kg>10.5;
-
 SELECT name FROM animals WHERE neutered IS TRUE;
-
 SELECT name FROM animals WHERE name NOT IN ('Gabumon');
-
 SELECT name FROM animals WHERE weigth_kg BETWEEN 10.4 and 17.3;
+
+/*Vet clinic database: query and update animals table*/
+
+BEGIN;
+UPDATE animals SET species='unspecified';
+SELECT * FROM animals;
+ROLLBACK;
+
+BEGIN;
+UPDATE animals SET species='digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species='pokemon' WHERE species IS NULL;
+COMMIT;
+
+BEGIN;
+DELETE FROM animals;
+ROLLBACK;
```

Comment:
- [ ] Kindly verify the change was undone 👍🏽

Reply:
![image](https://user-images.githubusercontent.com/114632980/230423254-eea85e91-e816-469c-ad5b-a710593db111.png)

Reply:
I am adding it to the Pull request description

Reply:
- [ ] @Feliverse as previously requested kindly add `SELECT species from animals;`  to verify the change was undone after rollback. Although it's shown in the screenshot you also need to add it to the queries.sql file, this will enable the user to run your code and get the same result.